docs/getting-started/codacy-quickstart.md

```diff
@@ -6,6 +6,8 @@ description: Set up Codacy to automatically analyze your source code and identif
 
 Codacy is an automated code quality and coverage platform that analyzes your source code and identifies issues as you go, helping your team ship robust software by scanning over 40 programming languages, such as JavaScript, Python, Java, C#, and PHP.
 
+Codacy uses multiple tools to analyze the source code of a program without executing it, looking for code patterns. Each of these patterns matches one or more well-known issues, from simple typos to critical security risks. This type of code inspection is called **static analysis**.
```

Comment:
This bit might work here in the introductory section rather than further on. Thoughts?